src/manifest.json

```diff
@@ -9,42 +9,396 @@
   "children": [
     {
       "type": "group",
-      "id": "redis_stack",
-      "label": "Redis Stack",
+      "id": "ds",
+      "label": "Data structures explained",
+      "args": {
+        "initialIsOpen": false
+      },
+      "children": [
+        {
+          "type": "group",
+          "id": "ds-json",
```

Comment:
it is possible to add a summary per tutorial to briefly describe what this tutorial is.
the syntax is: “summary”: “{text}”,

it is better to keep the summary short enough - up to 100 characters to display it correctly in RedisInsight.

examples of the summary:
<img width="449" alt="image" src="https://github.com/RedisInsight/Tutorials/assets/99594890/645a1542-31d9-4479-92ec-7624e2e45d1b">

Reply:
It's probably better that you and/or Olga add these summaries. I'm happy to review once you've done so.